doc/requirements/sequential_cutoff

```diff
@@ -0,0 +1,164 @@
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting sequential cutoff promotion count for core
+id: promotion_count_core
+---
+
+The management tool shall allow to specify sequential cutoff promotion count
+for given core.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: >-
+  Setting sequential cutoff promotion count for all cores in given
+  cache instance
+id: promotion_count_all_cores
+---
+
+The management tool shall allow to specify sequential cutoff promotion count
+for all cores associated with given cache instance at once.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Allowed sequential cutoff promotion count values
+id: promotion_count_values
+---
+
+It shall be possible to specify sequential cutoff promotion count value
+in range from 0 to 65535 inclusive.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Default sequential cutoff promotion threshold value
+id: promotion_count_default
+---
+
+The default sequential cutoff promotion threshold shall be 8.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Loading sequential cutoff configuration on cache load
+id: loading_configuration
+---
+
+All sequential cutoff parameters shall be loaded from cache metadata for each
+core on cache load.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Sequential cutoff operation
+id: operation
+---
+
+The sequential cutoff shall detect stream of sequential io requests and when
+amount of data within the stream reaches the cutoff threshold, each next io
+request which continues the stream shall be handled in Pass-Through mode.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Detecting multiple sequential cutoff streams
+id: detecting_multiple_streams
+---
+
+The sequential cutoff shall detect at least 64 separate streams of sequential
+io requests.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Sequential cutoff for reads and writes
+id: reads_and_writes
+---
+
+The sequential cutoff shall track streams of read and write io requests
+separately.
```

Comment:
1. `IO request`
2. `The system shall`

Reply:
1. Fixed.
2. I'm not sure if being that generic is a good thing. I would not obfuscate requirements for sake of it.